app/src/main/java/com/rahobbs/todo/helpers/TodoLab.java

```diff
@@ -0,0 +1,119 @@
+package com.rahobbs.todo.helpers;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.rahobbs.todo.database.TodoBaseHelper;
+import com.rahobbs.todo.database.TodoCursorWrapper;
+import com.rahobbs.todo.database.TodoSchema.TodoTable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Created by rachael on 2/2/16.
+ * Creates an array list of the items to display
+ */
+public class TodoLab {
```

Comment:
Lab is another big nerd ranch word that I might avoid... it doesn't really mean anything and it's very conflated. When naming I'd start with the longest most specific name you can. then you can see if a class is doing too much and also you can make sure you're always naming things accurately and not just using vague words like `manager` or `helper` or `lab`